### PR TITLE
zh-CN: tutorials/keras/basic_text_classification_with_tfhub.ipynb

### PR DESCRIPTION
**Pull request by Berrywrq**

### CODE REVIEW

**Comment by gaoljhy** [comment]
ok,lgtm

**Comment by Berrywrq** [comment]
It seems still can't pass the Ubuntu Sanity Check and I have no idea to solve this issue, could you help me to ask some suggestions from reviewers group or @Mr-Linus? Thanks!

File: site/zh-cn/beta/tutorials/keras/basic_text_classification_with_tfhub.ipynb [hunk 1]
```diff
@@ -0,0 +1,471 @@
...
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "zXXx5Oc3pOmN"
+   },
+   "outputs": [],
+   "source": [
+    "# Split the training set into 60% and 40%, so we'll end up with 15,000 examples\n",
```
Comment (Mr-Linus) [inline site/zh-cn/beta/tutorials/keras/basic_text_classification_with_tfhub.ipynb]: 把这个地方的注释翻译一下就可以了，注意中英文之间使用空格分隔

**Comment by Mr-Linus** [comment]
The CI checks that the article need to use "you" instead of "we" so you might translate line 168 into Chinese and the CI will be passed.

**Comment by Mr-Linus** [comment]
ref : https://source.cloud.google.com/results/invocations/d4aec638-c5d6-4b88-a105-81f4972731c7

File: site/zh-cn/beta/tutorials/keras/basic_text_classification_with_tfhub.ipynb (hunk 1 above)
Comment (Berrywrq) [reply site/zh-cn/beta/tutorials/keras/basic_text_classification_with_tfhub.ipynb]: Thanks for your help!

Comment (Mr-Linus) [reply site/zh-cn/beta/tutorials/keras/basic_text_classification_with_tfhub.ipynb]: My pleasure. 😆

**Comment by lamberta** [comment]
Thanks for investigating. Yeah, the "you" vs "we" comes from our English style guide: https://developers.google.com/style/person
So the source docs is wrong here. Probably shouldn't run this check on translated notebooks, though.

**Comment by gaoljhy** [comment]
> Thanks for investigating. Yeah, the "you" vs "we" comes from our English style guide: https://developers.google.com/style/person
> So the source docs is wrong here. Probably shouldn't run this check on translated notebooks, though.

thanks,Learn more

**Pull request merged**